File: test/dotnet/Integration/RedisListTriggerTestFunctions/ListTrigger_RightDirection.cs
using Microsoft.Extensions.Logging;

namespace Microsoft.Azure.WebJobs.Extensions.Redis.Tests.Integration
{
    public static class ListTrigger_RightDirection
    {
        [FunctionName(nameof(ListTrigger_RightDirection))]
        public static void Run(
            [RedisListTrigger(IntegrationTestHelpers.ConnectionStringSetting, nameof(ListTrigger_RightDirection), IntegrationTestHelpers.PollingIntervalLong, listDirection: ListDirection.RIGHT)] string entry,
            ILogger logger)
        {
            logger.LogInformation(IntegrationTestHelpers.GetLogValue(entry));
        }
    }
}

File: test/java/src/main/java/com/function/RedisListTrigger/ListTrigger_RightDirection.java
package com.function.RedisListTrigger;

import com.function.Common;
import com.microsoft.azure.functions.*;
import com.microsoft.azure.functions.annotation.*;
import com.microsoft.azure.functions.redis.annotation.*;

public class ListTrigger_RightDirection {
    @FunctionName("ListTrigger_RightDirection")
    public void run(
            @RedisListTrigger(
                name = "req",
                connectionStringSetting = Common.ConnectionString,
                key = "ListTrigger_RightDirection",
                listDirection = ListDirection.RIGHT)
                String message,
            final ExecutionContext context) {
            context.getLogger().info(message);
    }
}
